SConstruct

```diff
@@ -421,6 +421,9 @@ if enscons_active:
     # and so we let it determine the wheel tag by itself.
     env['ROOT_IS_PURELIB'] = False
 
+    # This enables tagging the wheel with the limited api, and also sets the target python version.
+    env["LIMITED_API_TARGET"] = (3, 5)
```

Comment:
Could you add a maintainer note that if the version specified here is changed then the `#define Py_LIMITED_API` line also needs to be updated?

Reply:
(This is a comment on an older version of the file, btw)

Reply:
Solved this in a different way, by having both these bits be pulled from a single source of truth